opentelemetry-otlp/src/exporter/mod.rs

```diff
@@ -195,6 +224,17 @@ pub trait WithExportConfig {
     fn with_timeout(self, timeout: Duration) -> Self;
     /// Set export config. This will override all previous configuration.
     fn with_export_config(self, export_config: ExportConfig) -> Self;
+    /// Set insecure connection. Disable TLS
+    fn with_insecure(self) -> Self;
+    /// Set the certificate file to validate the OTLP server connection
+    /// This is only available when the `tls` feature is enabled.
+    fn with_certificate<T: Into<String>>(self, certificate: T) -> Self;
```

Comment:
We can gate this function under `#[cfg(feature = "tls")]`?

Reply:
Done!